arelle/ViewFile.py

```diff
@@ -8,12 +8,10 @@
 from lxml import etree
 from decimal import Decimal
 from arelle.FileSource import FileNamedStringIO
-if sys.version[0] >= '3':
-    csvOpenMode = 'w'
-    csvOpenNewline = ''
-else:
-    csvOpenMode = 'wb' # for 2.7
-    csvOpenNewline = None
+
+csvOpenMode = 'w'
+csvOpenNewline = ''
```

Comment:
For these single use globals that only existed to gloss over 2-vs-3, it seems like it would be clearer to inline the values and remove the values altogether.